<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>540</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>로그인</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="streamer_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>490</y>
     <width>108</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>  streamer</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="view_btn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>490</y>
     <width>108</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>viewer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>430</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>닉네임:</string>
   </property>
  </widget>
  <widget class="QLabel" name="img_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>641</width>
     <height>391</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
